jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jps.java

```diff
@@ -21,7 +21,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-package openj9.tools.attach.diagnostics;
+package openj9.tools.attach.diagnostics.tools;
```

Comment:
This will require the launcher scripts to be updated.

Reply:
Thanks for pointing that out.  I have been updating and testing the launcher scripts.